tap_gitlab/utils.py

```diff
@@ -29,7 +30,7 @@ def wrapper(*args, **kwargs):
                 timer = threading.Timer(every, semaphore.release)
                 timer.setDaemon(True)  # allows the timer to be canceled on exit
                 timer.start()
-                return result
+            return result
```

Comment:
Actually, can you please rip out this utils module and replace any references to it with references to the `singer` lib?